src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import ReactDOM from "react-dom";
+//component file
+import TodoContainer from "./components/TodoContainer";
+
+//stylesheet
+import "./App.css";
+
+ReactDOM.render(<React.StrictMode><TodoContainer /></React.StrictMode>, document.getElementById("root"));
```

Comment:
- [ ] Kindly make sure to use createRoot instead.ReactDOM.render which is no longer supported in React 18. Until you switch to the new API, your app will behave as if it's running React 17.